src/test/java/CodeBuildActionTest.java

```diff
@@ -92,4 +104,23 @@ public void testFormatPhaseHistoryFinalTransition() {
         assert(r.get(2).getPhaseStatus().equals("SUCCEEDED"));
         assert(r.get(2).getDurationInSeconds().equals(0L));
     }
+
+    /**
+     * Ensures that the action with build phases can be serialized to the disk.
```

Comment:
Really need `RestartableJenkinsRule` to verify that it _worked_, but this at least verifies that `save` throws no exception, so OK.